theatres/anizay-2023/TCO30_Matias_TrapdoorSolution.tem_anizay/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,27 @@
+A socio-economical crisis has been brewing in the region of Anizay, and as a result, a civil war ensued. NATO had been assisting its allies and associates in evacuations in the weeks leading up to the major pullout, but now all that is left is some remnants of a few UNA forces still stationed at FOB Nauzad and FOB Obeh, while those who missed the evacuations have been left behind scrambling for a way to escape. As of recently the insurgency has proven to be a force to be reckoned with after carrying out several succesful attacks against the military detachments present in the region. As civilians scrambled to evacuate, insurgents made sure to attack Landay airfield in an attempt to hold as many of the population within the region as bargaining chips. In response to this, humanitarian efforts and foreign military involvement has quickly escalated prominently.
```

Comment:
I know this is probably from the template but still should be fixed, ideally on the template too...
```suggestion
A socio-economical crisis has been brewing in the region of Anizay, and as a result, a civil war ensued. NATO had been assisting its allies and associates in evacuations in the weeks leading up to the major pullout, but now all that is left is some remnants of a few UNA forces still stationed at FOB Nauzad and FOB Obeh, while those who missed the evacuations have been left behind scrambling for a way to escape. As of recently the insurgency has proven to be a force to be reckoned with after carrying out several successful attacks against the military detachments present in the region. As civilians scrambled to evacuate, insurgents made sure to attack Landay airfield in an attempt to hold as many of the population within the region as bargaining chips. In response to this, humanitarian efforts and foreign military involvement has quickly escalated prominently.
```